values.yaml

```diff
@@ -843,7 +843,7 @@ puppetdb:
     #  service.kubernetes.io/ibm-load-balancer-cloud-provider-ip-type: "private"
     labels: {}
     loadBalancerIP: ""
-    clusterIP: ""
+    # clusterIP: "8.9.9.8"
```

Comment:
this is a nitpick but please use IP addresses that are reserved for documentation purposes - https://datatracker.ietf.org/doc/html/rfc5737